doc/stages/filters.hag.rst

```diff
@@ -172,3 +188,8 @@ allow_extrapolation
     is false), extrapolation is used to assign the ``HeightAboveGround``
     value.
     [Default: false]
+
+respect_ground_classification
```

Comment:
This name is way too long, IMO.  Its behavior needs to be applied to all processing modes.

Reply:
Yes, but ground points are already 0 by definition in the other two modes because HAG is only calculated for non-ground points.

Reply:
We zero HAG for ground in those modes.  If we're going to do this, I think it should be consistent.  Either you ignore ground points altogether, or you set to 0.

Reply:
I'm not sure I understand @abellgithub - By default this mode is zeroing HAG for ground nodes, the option just allows you to opt out.

Any ideas for a better/shorter name? I'm happy with `zero_ground` or maybe `ignore_classification` (but default to false and the logic is backwards)?